src/platform/forms-system/src/js/components/FieldTemplate.jsx

```diff
@@ -47,32 +47,23 @@ export default function FieldTemplate(props) {
   let errorSpanId;
   let errorSpan;
   let errorClass;
-  // let errorStyle = 'none';
+
   if (hasErrors) {
     errorClass = isDateField ? 'input-error-date' : 'usa-input-error';
     errorSpanId = `${id}-error-message`;
-    // errorStyle = 'block';
     errorSpan = (
       <span className="usa-input-error-message" role="alert" id={errorSpanId}>
         <span className="sr-only">Error</span> {rawErrors[0]}
       </span>
     );
   }
 
-  /*
-  if (environment.isStaging()) {
-    errorSpan = (
-      <span
-        style={{ display: errorStyle }}
-        className="usa-input-error-message"
-        role="alert"
-        id={errorSpanId}
-      >
-        <span className="sr-only">Error</span> {rawErrors[0]}
-      </span>
-    );
+  if (environment.isDev()) {
+    let errorSpanClass = classNames('usa-input-error-message', {
```

Comment:
I think you're missing the definition of `errorSpan` that uses this class.

Reply:
it's right below
```    errorSpan = (
      <span className={errorSpanClass} role="alert" id={errorSpanId}>
        <span className="sr-only">Error</span> {rawErrors[0]}
      </span>
    );```